runtime/backtrace_byt.c

```diff
@@ -266,6 +266,11 @@ int caml_alloc_backtrace_buffer (void)
   return 0;
 }
 
+void caml_free_backtrace_buffer(backtrace_slot *backtrace_buffer) {
```

Comment:
The allocation code above is still using `sizeof(code_t)`, should it be switched to `bactrace_slot` too?

Reply:
Currently, `code_t` is defined as `opcode_t *`, and `backtrace_slot` as `void *` (both in caml/misc.h). Using `sizeof` on either ones gives the same result as they are both pointer types.

I think the code may be nicer if `backtrace_slot` was defined conditionally depending on whether we are in native or bytecode mode, so that it is *synonymous* to `code_t` in this file. In this alternative world, the change you suggest would not be necessary.

In the context of the present PR, we went for the simplest modification that worked without forcing us to write two different prototypes for the function. So we left `code_t` everywhere else in this file. Does that make sense?